<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Post-transcriptional processing events</title>
    <meta charset="utf-8" />
    <meta name="author" content="Alistair Bailey" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/panelset/panelset.css" rel="stylesheet" />
    <script src="libs/panelset/panelset.js"></script>
    <script src="libs/xaringanExtra-webcam/webcam.js"></script>
    <script id="xaringanExtra-webcam-options" type="application/json">{"width":"450","height":"337.5","margin":"1em"}</script>
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <link rel="stylesheet" href="assets/xaringan-themer.css" type="text/css" />
    <link rel="stylesheet" href="assets/xaringan-extra.css" type="text/css" />
    <link rel="stylesheet" href="assets/uos.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: hide-slide-number, middle, center, title-slide

# Post-transcriptional processing events
### Alistair Bailey
### 22nd September 2021

---





class: hide-count

## .marine-1[Post-transcriptional events]

.pull-left.marine-1[In eukaryotes this refers to processing events that follow transcription of genetic DNA to messenger RNA, and prior to mRNA translation into protein. 
&lt;br&gt;&lt;br&gt;
These events are necessary for export of mRNA from the nucleus to the cytoplasm and subsequent translation into protein.]

.pull-right[&lt;img src="img/post-transcriptional-events.png" alt="post txn events" style="width:80%;height:80%;"&gt;]

---
class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1 animated flipInY"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;
---

class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2 animated flipInX"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

---

class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile splicing animated flipInY"&gt;&lt;div&gt;Splicing&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

---

class: full-screen hide-count middle end-mod-1

#.center[End modification:&lt;br&gt;5' Cap]

---

class: hide-count end-mod-1

## .white[End modification: 5' Cap]

.pull-left[### Why?
+ Identifies mRNA
+ Helps export from nucleus
+ Protects against degradation
+ Initiation of protein synthesis
]

&lt;!-- .tiny[By Zephyris - English Wikipedia, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=1379696]] --&gt;
      
.pull-right.bg-white.center[&lt;img src="img/5'_cap_structure.png" alt="End Cap" style="width:100%;height:100%;"&gt;]
.pull-right.tiny[[Zephyris (2005) Wikipedia](https://commons.wikimedia.org/w/index.php?curid=1379696)]

---

class: hide-count 

.center[&lt;img src="img/capping_steps.jpeg" alt="Cap steps" style="width:85%;height:85%;"&gt;]

.bottom.center.tiny[Ramanathan et. al., Nucleic Acids Res (2016) https://doi.org/10.1093/nar/gkw551]

???

Enzymatic steps involved in RNA capping. The RNA triphosphatase activity (TPase) removes the γ-phosphate from 5′ triphosphate, generating a diphosphate 5′ end and inorganic phosphate (reaction [1]). The guanylyltransferase (GTase) activity consumes a GTP molecule and forms a covalent intermediate containing with a lysyl-Nζ-5′-phosphoguanosine (reaction [2.1]). In the presence of a 5′ diphosphate RNA, the GTase activity transfers the 5′-phosphoguanosine (GMP) to the 5′ diphosphate, forming a 5′-5′ triphosphate linkage between the first base of the RNA and the capping base (reaction [2.1]). In the presence of S-adenosylmethionine (SAM), the guanine-N7 methyltransferase (MTase) activity adds a methyl group to N7 amine of the guanosine cap to form the cap 0 structure (reaction [3]). Finally, the m7G cap-specific 2′O MTase modifies the 2′O of +1 ribose and generates the cap 1 structure (reaction [4]).

---

class: full-screen hide-count middle end-mod-2

#.center[End modification:&lt;br&gt;Polyadenylation]

---

class: hide-count end-mod-2

## End modification: Polyadenylation

Next section

---

class: full-screen hide-count middle splicing

#.center[Splicing]

---

class: hide-count middle splicing

.center[
&lt;video muted autoplay width="85%" height="85%"&gt; 
&lt;source src="animation/3dsplicingaudiotitle.mp4" type="video/mp4"&gt;
Your browser does not support the video tag.
&lt;/video&gt;
]

.center[[DNA Learning Centre, Cold Spring Harbour Laboratory](https://dnalc.cshl.edu/resources/3d/rna-splicing.html)]

???

As DNA is transcribed into RNA it needs to be edited to remove non-coding regions, or introns, shown in green. This editing process is called splicing, which involves removing the introns, leaving only the yellow, protein-coding regions, called exons.

RNA splicing begins with assembly of helper proteins at the intron/exon borders. These splicing factors act as beacons to guide small nuclear ribo proteins to form a splicing machine, called the spliceosome. The animation is showing this happening in real time. The spliceosome then brings the exons on either side of the intron very close together, ready to be cut. One end of the intron is cut and folded back on itself to join and form a loop. The spliceosome then cuts the RNA to release the loop and join the two exons together. The edited RNA and intron are released and the spliceosome disassembles.

This process is repeated for every intron in the RNA. Numerous spliceosomes, shown here in purple, assemble along the RNA. Each spliceosome removes one intron, releasing the loop before disassembling. In this example, three introns are removed from the RNA to leave the complete instructions for a protein.

---

class: hide-count middle

&lt;div class="grid-3-1a"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;br&gt;&lt;br&gt;Bullet&lt;br&gt;Bullet&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile splicing"&gt;&lt;div&gt;Splicing&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

---

class: hide-count

#.marine-1[Resources]


```
## Loading required namespace: bibtex
```

- 
&lt;p&gt;&lt;cite&gt;Ramanathan, A., G. B. Robb, and S. Chan
(2016).
&amp;ldquo;mRNA capping: biological functions and applications&amp;rdquo;.
In: &lt;em&gt;Nucleic Acids Research&lt;/em&gt;.
URL: &lt;a href="https://doi.org/10.1093/nar/gkw551"&gt;https://doi.org/10.1093/nar/gkw551&lt;/a&gt;.&lt;/cite&gt;&lt;/p&gt;
- 
&lt;p&gt;&lt;cite&gt;Neve, J., R. Patel, Z. Wang, et al.
(2017).
&amp;ldquo;Cleavage and polyadenylation: Ending the message expands gene regulation&amp;rdquo;.
In: &lt;em&gt;RNA Biology&lt;/em&gt;.
URL: &lt;a href="https://doi.org/10.1080/15476286.2017.1306171"&gt;https://doi.org/10.1080/15476286.2017.1306171&lt;/a&gt;.&lt;/cite&gt;&lt;/p&gt;
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
