<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Post-transcriptional processing events</title>
    <meta charset="utf-8" />
    <meta name="author" content="Alistair Bailey" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/panelset/panelset.css" rel="stylesheet" />
    <script src="libs/panelset/panelset.js"></script>
    <script src="libs/xaringanExtra-webcam/webcam.js"></script>
    <script id="xaringanExtra-webcam-options" type="application/json">{"width":"450","height":"337.5","margin":"1em"}</script>
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <link rel="stylesheet" href="assets/xaringan-themer.css" type="text/css" />
    <link rel="stylesheet" href="assets/xaringan-extra.css" type="text/css" />
    <link rel="stylesheet" href="assets/uos.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: hide-slide-number, middle, center, title-slide

# Post-transcriptional processing events
### Alistair Bailey
### 22nd September 2021

---




class: hide-count

## .marine-1[Post-transcriptional events]

.pull-left.marine-1[In eukaryotes this refers to processing events that follow transcription of genetic DNA to messenger RNA, and prior to mRNA translation into protein. 
&lt;br&gt;&lt;br&gt;
These events are necessary for export of mRNA from the nucleus to the cytoplasm and subsequent translation into protein.]

.pull-right[&lt;img src="img/post-transcriptional-events.png" alt="post txn events" style="width:80%;height:80%;"&gt;]

???

Post-transcriptional processing here refers to events that follow the transcription of genetic DNA into messanger RNA, and prior to mRNA translation into protein.

I'll be limiting the scope to events common in eurkaryotes, but I've provided a review on post-transcriptional regulation in prokaryotes at the end in the resources for you to investigate and compare.

On the right here then, we have a generic nucleated cell, and I've highlighted the nucleus which is where the post-transcriptional processes are occuring. 

The learning objectives for this lecture are to understand the three most important post-transcriptional processing events that are necessary for mRNA export to the cytoplasm and subsequent translation into protein.

---
class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1 animated flipInY"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

???

These three processes are:

+ The modification of the 5' end of mRNA, the so-called 5' cap
---

class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2 animated flipInX"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

???

These three processes are:

+ The modification of the 5' end of mRNA, the so-called 5' cap
+ The modification of the 3' end of mRNA: a process called polyadenylation.
---

class: full-screen hide-count middle

&lt;div class="grid-3-1"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile splicing animated flipInY"&gt;&lt;div&gt;Splicing&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

???

These three processes are:

+ The modification of the 5' end of mRNA, the so-called 5' cap
+ The modification of the 3' end of mRNA: a process called polyadenylation.
+ And the process of splicing of mRNA.

---

class: full-screen hide-count middle end-mod-1

#.center[End modification:&lt;br&gt;5' Cap]

???

Beginning with the 5' cap end modification.

---

class: hide-count end-mod-1

## .white[End modification: 5' Cap]

.pull-left.bg-white.center[&lt;img src="img/5'_cap_structure.png" alt="End Cap" style="width:90%;height:90%;"&gt; &lt;br&gt; [Cap 2: Zephyris (2005) Wikipedia](https://commons.wikimedia.org/w/index.php?curid=1379696)]

.pull-right[
### 7-methlyguanosine

### Why?
+ Identifies mRNA
+ Helps export from nucleus
+ Protects against degradation
+ Initiation of protein synthesis
]

&lt;!-- .tiny[By Zephyris - English Wikipedia, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=1379696]] --&gt;
      
???

On the left we can see the final structure of the 5' prime cap, which is a 7-methlyguanosine molecule attached by three phosphates to the 5' end of the mRNA.

The cap serves various functions:

+ It identifies the RNA as mRNA
+ It helps export the mRNA from the nucleus
+ It protects mRNA against degradation
+ And it's involved in the initiation of protein synthesis.

Again I've included a more detailed review of the 5' cap, including it's functions, in the resources at the end.

But let's look in more detail at how the capping process works. And if I was a student worrying about exam questions, I'd pay close attention to the next slide.


---

class: hide-count middle

.center[&lt;img src="img/capping_steps_short.jpg" alt="Cap steps" style="width:100%;height:100%;"&gt;]

.bottom.center.tiny[Ramanathan et. al., Nucleic Acids Res (2016) https://doi.org/10.1093/nar/gkw551]

???

Ok, this is a complicated slide. What we need to notice here is that there are:

+ 3 enzymes are involved in the formation of the 7-methlyguanosine cap. These are indicated in the middle of the reaction equations in their contracted form.
+ And we have other molecules that are consumed/produced during cap formation
+ And going from top left to bottom right is a step in the formation of the cap involving one of the enzymes leading to what is called the cap 0 structure.

(i) RNA triphosphatase (TPase) removes the γ-phosphate from 5′ triphosphate, generating a diphosphate 5′ end and inorganic phosphate (reaction [1]). 

(ii) Meanwheile guanylyltransferase (GTase) consumes a GTP molecule and forms a covalent intermediate with a lysyl-N-5′-phosphoguanosine (reaction [2.1]), releasing a pyrophospate molecule from GTP.

(iii) Guanylyltransferase (GTase) activity transfers the 5′-phosphoguanosine (GMP) from the intermediate to the 5′ diphosphate RNA, forming a 5′-5′ triphosphate linkage between the first base of the RNA and the capping base (reaction [2.1]). 

(iv) In the presence of S-adenosylmethionine (SAM), the methyltransferase (MTase) adds a methyl group to N7 amine of the guanosine cap to form the cap 0 structure (reaction [3]) of
7-methlyguanosine.

---

class: hide-count end-mod-1

## .white[End modification: 5' Cap]

.bg-white.center[&lt;img src="img/5'_cap_structure.png" alt="End Cap" style="width:45%;height:45%;"&gt; &lt;br&gt; [Zephyris (2005) Wikipedia](https://commons.wikimedia.org/w/index.php?curid=1379696)]

???

An additional methyl group may be added to the +1 and +2 ribonucleotides by another methyltransferase leading to the cap 2 structure we saw originally.

---

class: full-screen hide-count middle end-mod-2

#.center[End modification:&lt;br&gt;Polyadenylation]

???

Following 5' cap, the other end of the mRNA is also modified in a process called polyadenylation.

This process adds a tail of adenosines (A-DEN-O-SIN) to the 3' end of mRNA. This is part of the termination of transcription and aids in the nuclear export process. The poly(A) tail also protects the mRNA molecule from enzymatic degradation in the cytoplasm prior to translation.

A detailed review is provided in the resources, but let's look at a simplified view of the process itself.

---

class: hide-count end-mod-2 middle

&lt;!-- .pull-left[## End modification: Polyadenylation
+ CFI and CFII: cleavage factors I and II
+ CPSF: cleavage and polyadenylation specificity factor
+ Poly A pol: Poly A polymerase
+ CstF: cleavage stimulation factor
+ PAB2: Polyadenylate-binding protein 2
]

.pull-right --&gt;

.center[&lt;img src="img/PolyAv3.png" alt="PolyA" style="width:100%;height:100%;"&gt;]

???

Another complicated slide, so let's break it down. 

+ At the top we have our mRNA with it's 5' cap. Towards the 3' end we have specific nucleotide sequences to which mRNA cleavage factors bind, and the poly A polymerase bind.

+ Upon binding of the cleavage factors to the mRNA, cleavage of the 3' end is catalysed by poly(A) polymerase and the factors CPSF and CstF.

+ Poly A polymerase bound then begins to add adenine bases.

+ Another protein PAB2 binds to the tail, increasing the polyemerase affinity which continues adding nucleotides until about 200-250 have been added, at which length the CPSF interaction is interupted and polyadenylation stops.

At this point, both ends of the mRNA have been modified.

---

class: full-screen hide-count middle splicing

#.center[Splicing]

???

Following end modifications, the third post-transcriptional process to consider is splicing, a word with naval origins. 

One form of splicing being where a rope is split, parts removed, and then the remaining parts of the rope joined back together.

This is the type of splicing under consideration here.

---

class: hide-count middle splicing

.center[
&lt;video muted controls width="85%" height="85%"&gt; 
&lt;source src="animation/3dsplicingaudiotitle.mp4" type="video/mp4"&gt;
Your browser does not support the video tag.
&lt;/video&gt;
]

.center[[DNA Learning Centre, Cold Spring Harbour Laboratory](https://dnalc.cshl.edu/resources/3d/rna-splicing.html)]

???

As DNA is transcribed into RNA it needs to be edited to remove non-coding regions, or introns, shown in green. This editing process is called splicing, which involves removing the introns, leaving only the yellow, protein-coding regions, called exons.

RNA splicing begins with assembly of helper proteins at the intron/exon borders. These splicing factors act as beacons to guide small nuclear ribo proteins to form a splicing machine, called the spliceosome. The animation is showing this happening in real time. The spliceosome then brings the exons on either side of the intron very close together, ready to be cut. One end of the intron is cut and folded back on itself to join and form a loop. The spliceosome then cuts the RNA to release the loop and join the two exons together. The edited RNA and intron are released and the spliceosome disassembles.

This process is repeated for every intron in the RNA. Numerous spliceosomes, shown here in purple, assemble along the RNA. Each spliceosome removes one intron, releasing the loop before disassembling. In this example, three introns are removed from the RNA to leave the complete instructions for a protein.

---

class: hide-count splicing

## Alternative Splicing

.center[&lt;img src="img/DNA_alternative_splicing.gif" alt="Alt-Splice" style="width:85%;height:85%;"&gt;&lt;br&gt;National Human Genome Research Institute]

???

If this process does not lead to splicing all the available exons in the mRNA together, it's called alternative splcing. 
As shown in the examples here where alternative splicing produces three protein isoforms. Protein A includes all of the exons, whereas Proteins B and C result from exon skipping.

---

class: hide-count middle

&lt;div class="grid-3-1a"&gt;
&lt;div class="extension-tile end-mod-1"&gt;&lt;div&gt;End modification:&lt;br&gt;5' Cap&lt;br&gt;&lt;br&gt;The addition of 7 methyl-guanosine to the 5’ end of mRNA&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile end-mod-2"&gt;&lt;div&gt;End modification:&lt;br&gt;Polyadenylation&lt;br&gt;&lt;br&gt;The addition of adenosines to 3' end of mRNA&lt;/div&gt;&lt;/div&gt;
&lt;div class="extension-tile splicing"&gt;&lt;div&gt;Splicing&lt;br&gt;&lt;br&gt;The removal of introns and joining of exons&lt;/div&gt;&lt;/div&gt;
&lt;/div&gt;

???

To re-cap. We had three learning objectives:

+ The 5' end cap: the addition of 7 methyl-guanosine to the 5’ end of RNA. 
+ Polyadenylation: the addition of ~200 adenosines to 3' tail of RNA
+ Splcing: removal of introns and joining of exons to create a protein coding mRNA for translation. 

Following this three events, mRNA can be exported to the cytoplasm for translation into protein via the nuclear pore complex.

In this lecture we've only considered this processes in terms of how they are supposed to work in a canonical form. That is to say in a general sense. But as with much of biology, there are alternative forms of these processes and less common events. We've touched upon alternative splicing, but there is also alternative polyadenylation. Although it's beyond the scope of this lecture, it is important to consider what happens when processes malfunctions or rare events such as mRNA editiing occur, especially in the context of disease.

---

class: hide-count

##.marine-1[Resources]



- 
&lt;p&gt;&lt;cite&gt;Picard, F., C. Dressaire, L. Girbal, et al.
(2009).
&amp;ldquo;Examination of post-transcriptional regulations in prokaryotes by integrative biology&amp;rdquo;.
En.
In: &lt;em&gt;Comptes Rendus Biologies&lt;/em&gt; 332.11.
URL: &lt;a href="https://doi.org/10.1016/j.crvi.2009.09.005"&gt;https://doi.org/10.1016/j.crvi.2009.09.005&lt;/a&gt; (visited on Sep. 21, 2021).&lt;/cite&gt;&lt;/p&gt;
- 
&lt;p&gt;&lt;cite&gt;Ramanathan, A., G. B. Robb, and S. Chan
(2016).
&amp;ldquo;mRNA capping: biological functions and applications&amp;rdquo;.
In: &lt;em&gt;Nucleic Acids Research&lt;/em&gt;.
URL: &lt;a href="https://doi.org/10.1093/nar/gkw551"&gt;https://doi.org/10.1093/nar/gkw551&lt;/a&gt;.&lt;/cite&gt;&lt;/p&gt;
- 
&lt;p&gt;&lt;cite&gt;Neve, J., R. Patel, Z. Wang, et al.
(2017).
&amp;ldquo;Cleavage and polyadenylation: Ending the message expands gene regulation&amp;rdquo;.
In: &lt;em&gt;RNA Biology&lt;/em&gt;.
URL: &lt;a href="https://doi.org/10.1080/15476286.2017.1306171"&gt;https://doi.org/10.1080/15476286.2017.1306171&lt;/a&gt;.&lt;/cite&gt;&lt;/p&gt;
- 
&lt;p&gt;&lt;cite&gt;Elkon, R., A. P. Ugalde, and R. Agami
(2013).
&amp;ldquo;Alternative cleavage and polyadenylation: extent, regulation and function&amp;rdquo;.
En.
In: &lt;em&gt;Nature Reviews Genetics&lt;/em&gt; 14.7.
URL: &lt;a href="https://www.nature.com/articles/nrg3482"&gt;https://www.nature.com/articles/nrg3482&lt;/a&gt; (visited on Sep. 10, 2021).&lt;/cite&gt;&lt;/p&gt;
- 
&lt;p&gt;&lt;cite&gt;Zhang, Y., J. Qian, C. Gu, et al.
(2021).
&amp;ldquo;Alternative splicing and cancer: a systematic review&amp;rdquo;.
En.
In: &lt;em&gt;Signal Transduction and Targeted Therapy&lt;/em&gt; 6.1.
URL: &lt;a href="https://www.nature.com/articles/s41392-021-00486-7"&gt;https://www.nature.com/articles/s41392-021-00486-7&lt;/a&gt; (visited on Sep. 21, 2021).&lt;/cite&gt;&lt;/p&gt;

???

So here I'm provided some resources for you to start exploring some areas I haven't covered and non-canonical post-transcriptional events:

+ prokaryotic post-transcpitional regulation.
+ More details of mRNA end modifications.
+ And alternative poly(A) processes and alternative splicing in the context of cancer.

    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
